### PR TITLE
add CoreShop to e-commerce section

### PR DESCRIPTION
**Pull request by dpfaffenbauer**

### CODE REVIEW

**Comment by Kickball** [comment]
Please order alphabetically.

**Comment by dpfaffenbauer** [comment]
ok, done

**Comment by AndyR207** [comment]
You need a full stop after your description.

**Comment by dpfaffenbauer** [comment]
done

File: README.md [hunk 1]
```diff
@@ -400,6 +400,7 @@ _Recipe management_
 ### E-commerce
 
 * [Attendize](https://www.attendize.com/) - Ticket selling and event management platform. ([Demo](https://www.attendize.com/documentation.php#demo), [Source Code](https://github.com/attendize/attendize)) `AAL` `PHP`
+* [CoreShop](https://www.coreshop.org) - CoreShop is a e-commerce plugin for Pimcore. ([Source Code](https://github.com/coreshop/CoreShop)) `GPL` `PHP`
```
Comment (nodiscc) [inline README.md]: GPLv3

Comment (dpfaffenbauer) [inline README.md]: done

**Pull request merged**